packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -43,6 +50,38 @@ export function authorizationServiceBuilder(
         showUsers: isClientConsumer(client.data.consumerId, organizationId),
       };
     },
+
+    async createConsumerClient(
+      clientSeed: ApiClientSeed,
+      organizationId: TenantId,
+      correlationId: string,
+      logger: Logger
+    ): Promise<{ client: Client; showUsers: boolean }> {
+      logger.info(
+        `Creating CONSUMER client ${clientSeed.name} for consumer ${organizationId}"`
+      );
+      const client: Client = {
+        id: generateId(),
+        consumerId: organizationId,
+        name: clientSeed.name,
+        purposes: [],
+        description: clientSeed.description,
+        relationships: [],
+        kind: clientKind.consumer,
+        users: clientSeed.members.map(unsafeBrandId<UserId>),
+        createdAt: new Date(),
+        keys: [],
+      };
+
+      await repository.createEvent(
+        toCreateEventClientAdded(client, correlationId)
+      );
+
+      return {
+        client,
+        showUsers: client.consumerId === organizationId,
```

Comment:
In this case we are sure that the requester is the one who is creating the client, so this can be `true`.